### ISSUE TITLE
Change defaults for radio buttons to meet WCAG requirements

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/sun#977]
Currently, the default for a deselected radio button is to have 0.6 opacity. This causes most deselected radio buttons to fail contrast requirements.  Here are some examples:

<img width="108" height="117" alt="Image" src="https://github.com/user-attachments/assets/240982a8-90ae-4f99-a00b-c3ca86f13875" />

<img width="144" height="152" alt="Image" src="https://github.com/user-attachments/assets/da9b9318-584c-446a-9fa3-6dfdbc8ac887" />

<img width="396" height="101" alt="Image" src="https://github.com/user-attachments/assets/e5a0d120-e224-4ae4-963e-994271fc804c" />

Can we change the defaults values?

**Comment by marlitas** [comment]
@catherinecarter We would like you to lead the design on how to resolve this for rectangular radio buttons. This will probably require some research on what options exist and then confirmation with the design team. A developer will then implement the requested changes.

**Comment by catherinecarter** [comment]
@Nancy-Salpepi and I met to discuss this issue and learn more about needs and implications. Here is information we learned and questions we have:

**Learned**
* Many many sims use radioButton groups (we already knew this, but worth noting)
* This likely also impacts toggles (A/B switches) that have images on either side (e.g., Number Pairs, number line scene)
      <img width="150" height="50" alt="Image" src="https://github.com/user-attachments/assets/0805e8d5-4ddc-4958-86b4-74c06c7b53e1" />

* The issue is about the reduced opacity layer on top of the button that makes it look disabled
* It's possible to remove the layer, but the process in doing so is not efficient (CM's suggested code is below):
   `RectangularRadioButtonGroupOptions.radioButtonOptions.buttonAppearanceStrategyOptions.deselectedButtonOpacity`
   `RectangularRadioButtonGroupOptions.radioButtonOptions.contentAppearanceStrategyOptions.deselectedContentOpacity`
   See this example: https://github.com/phetsims/models-of-the-hydrogen-atom/blob/f1f8b69f52b02e4a1fba36923e78451dc569ad5a/js/common/view/AtomicModelRadioButtonGroup.ts#L44

* If the reduced opacity layer is removed, it might be possible to replace the layer with a wider stroke and a color acting as a highlight around the button (e.g., MotHA)
   <img width="200" height="244" alt="Image" src="https://github.com/user-attachments/assets/8bcde56d-6500-4f1d-9c99-4acbb5b98883" />

**Questions we Have**
* If an option to remove the layer is implemented Common Code, what are the consequences?
   * Would it better to have the default ON to have the layer there, or default OFF to not have the layer there?
   * How will this impact the many sims that use the layer to indicate not selected?
 
 
* Noting that the [guidelines for color in sims](https://docs.google.com/document/d/1rlVX9DHXclCtpcFV-5YAoA0uI0Ui_H1mzPJck7v8PcM/edit?tab=t.0#heading=h.w80wjto8jrbk) need to include this newly learned possible contrast failure

**Comment by pixelzoom** [comment]
It seems to me that a reasonable solution would be to simply remove the change in opacity that is causing the elements to look disabled.

In RectangularRadioButton:

```diff
-       deselectedButtonOpacity: 0.6
+       deselectedButtonOpacity: 1
-       deselectedContentOpacity: 0.6
+       deselectedContentOpacity: 1
```

In ABSwitch:

```diff
- // Uses opacity as the default method of indicating whether a {Node} label is {boolean} enabled.
- const DEFAULT_SET_LABEL_ENABLED = ( label: Node, enabled: boolean ) => {
-   label.opacity = enabled ? 1.0 : SceneryConstants.DISABLED_OPACITY;
- };
-      setLabelEnabled: DEFAULT_SET_LABEL_ENABLED,
+      setLabelEnabled: _.noop,
```

**Comment by jessegreenberg** [comment]
We pushed what @pixelzoom proposed in https://github.com/phetsims/sun/issues/977#issuecomment-4462779414. We also bumped the selected stroke for RectangularRadioButtonGroup from 1.5 -> 2, so that the "selected" button is a little easier to see.

Next, @Nancy-Salpepi and I are going to review these components in sims to make sure this addresses the issue, buttons still look good, and layout was not impacted. Blocking publication until this is reviewed.

We discussed how this would be a great place to work in a branch, if Launchpad were able to support it. Then it could be confirmed before disrupting main.

**Comment by Nancy-Salpepi** [comment]
I updated the publication and republication checklists to remind designers to double-check the radio buttons (and icons for AB toggles should also be checked) to see if additional design adjustments are needed. 

I am going to go ahead and close this.
